src/components/plots/EquilibriumPlot.tsx

```diff
@@ -33,30 +37,66 @@ const EquilibriumPlot: React.FC<PlotProps> = ({
 }) => {
     const { maxConcentration } = useContext(SimulariumContext);
 
-    const horizontalDottedLine = {
+    const hintOverlay = (
+        <div
+            style={{
+                position: "absolute",
+                width: 91,
+                fontSize: 11,
+                fontStyle: "italic",
+                textAlign: "center",
+                color: "#A0A0A0",
+                right: "50%",
+                top: "50%",
+                transform: "translate(50%, -50%)",
+            }}
+        >
+            Record point of equilibrium to plot a value
+        </div>
+    );
+
+    const lineOptions = {
+        color: AGENT_A_COLOR,
+        width: 0.5,
+        dash: "dot" as Dash,
+    };
+
+    const horizontalLine = {
         x: [0, kd * 2],
         y: [5, 5],
         mode: "lines",
-        line: {
-            color: GRAY_COLOR,
-            width: 1,
+        name: "50% bound",
+        hovertemplate: "50% bound",
+        hoverlabel: {
+            bgcolor: AGENT_A_COLOR,
         },
+        line: lineOptions,
+    };
+    const horizontalLineMax = {
+        x: [0, kd * 2],
+        y: [10, 10],
+        mode: "lines",
+        name: "Initial [A]",
+        hoverlabel: { bgcolor: AGENT_A_COLOR },
+        hovertemplate: "Initial [A]",
+        line: lineOptions,
     };
     const trace = [
-        horizontalDottedLine,
+        horizontalLine,
+        horizontalLineMax,
         {
             x,
             y,
             type: "scatter" as const,
             mode: "lines+markers" as const,
-            name: "equilibrium",
+            name: "",
```

Comment:
No name anymore?

Reply:
FWIW I'm not even sure what this `trace` var is supposed to be in the first place. What is this array type?

Reply:
it should actually be 'traces' I'll change that. it's a the group of individual datasets that all get plotted on one set of axis

Reply:
> No name anymore?

yeah, by default the name gets included in the hover info, so one way to avoid that is to make it an empty string. the other way is to include `<extra></extra>` in the hover template. I was ambivalent on which way was better